src/domain/area/guidelineByAreaFilter.tsx

```diff
@@ -0,0 +1,22 @@
+import { ChangeEventHandler } from 'react-router/node_modules/@types/react'
+
+interface props {
+  guidelineNameByArea: string
+  handleSearchChange: ChangeEventHandler
+}
+const GuidelineByAreaFilter = ({ handleSearchChange, guidelineNameByArea }: props): JSX.Element => {
+  return (
+    <div className="flex flex-row justify-center w-full mt-1 ">
+      <input
+        type="search"
+        className="text-center focus:text-disabled text-disabled mb-4 outline-none  border-b-2 border-black"
+        onChange={handleSearchChange}
+        value={guidelineNameByArea}
+        placeholder="Buscar Lineamiento"
+      />
+      <i className="icon-magnifier text-sm md:text-lg" />
```

Comment:
The magnifying glass doesn't seem to be aligned with the input. Lets add some padding or margin top so it moves a little bit down:
![image](https://user-images.githubusercontent.com/14185059/136863093-042d5e8e-6255-43bb-941e-7792ec89d58b.png)